bin/torii/src/main.rs

```diff
@@ -188,8 +190,11 @@ async fn main() -> anyhow::Result<()> {
     )
     .expect("Failed to start libp2p relay server");
 
-    info!(target: "torii::cli", "Starting torii endpoint: {}", format!("http://{}", args.addr));
-    info!(target: "torii::cli", "Serving Graphql playground: {}\n", format!("http://{}/graphql", args.addr));
+    let endpoint = format!("http://{}", args.addr);
+    let encoded = general_purpose::STANDARD.encode(&endpoint).to_string();
```

Comment:
i think we should url encode the address here, so it would be like:

`https://worlds.dev/torii?url=XYZ`

Reply:
i inplemented as `https://worlds.dev/torii/[encoded-url]` but url params also works 👍

Reply:
ok cool. url encoded is nicer i think, since a human can encode it easily. base64 is much harder